### ISSUE TITLE
How to remove version from output files?

### ISSUE THREAD

**Issue by cawa-93** [ProjectAliceDev/renpy-build#9]
I use this action to compile a game and upload the output files to the GitHub release . However, having a version complicates the process, as I can't have a static link to download the latest version of the game like `https://github.com/my/game/releases/latest/pc-game-title.zip`

Is there any way I can remove the version from the output file names?

**Comment by sr229** [comment]
I don't think we provide a filename override for this, so it's considered due diligence to update your title's artifact URLs on anywhere were it's defined.

Feel free to make a patch for that though!
